WISE-Community/WISE
# Allow teachers to copy and share run signup link with students

To test:
- Log in as a teacher
- Check that 'Share to Google Classroom' link has been removed from run menus on Class Schedule
- Verify that "Share with students" link appears for each run and clicking opens Share with Students dialog
- Review content of Share with Students dialog, verify that copying sign up/sign in link and run code works
- Verify that 'Share to Google Classroom' option appears in the dialog and functions properly when signed in to WISE with Google

Closes #2783.
